### PR TITLE
fix Compile Time Function Execution section

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: spec/function.dd [hunk 1]
```diff
@@ -2666,10 +2666,11 @@ void main()
...
 
-    $(P Functions which are both portable and free of global side-effects can be
-    executed at compile time. In certain contexts, such compile time execution
-    is guaranteed. It is called Compile Time Function Execution (CTFE) then.
-    The contexts that trigger CTFE are:)
+    $(P In contexts where a compile time value is required, functions
+    can be used to compute those values. This is called $(I Compile Time Function
+    Execution), or $(I CTFE).)
+
+    $(P These contexts include:)
```
Comment (aG0aep6G) [inline spec/function.dd:2673]: "Include" implies that there are other CTFE contexts not listed here. But the spec should describe the language completely, so the list here should be exhaustive.

Comment (atilaneves) [reply spec/function.dd:2673]: I agree with @aG0aep6G and think this should be changed to "these contexts are".

**Review by RazvanN7 — APPROVED** [review]

**Comment by RazvanN7** [comment]
@WalterBright mind the @aG0aep6G 's comment and this is good to go.

File: spec/function.dd (hunk 1 above)
Comment (WalterBright) [reply spec/function.dd]: done

**Pull request merged**